lang/python/python/Makefile

```diff
@@ -194,23 +194,17 @@ define Build/Compile
 endef
 
 define Build/InstallDev
-	$(INSTALL_DIR) $(1)/usr/include/ $(1)/usr/lib/ $(1)/usr/lib/pkgconfig
-	$(INSTALL_DIR) $(1)/usr/lib/python$(PYTHON_VERSION)/
+	$(INSTALL_DIR) $(1)/usr/include $(1)/usr/lib $(1)/usr/lib/pkgconfig
 	$(CP) \
 		$(PKG_INSTALL_DIR)/usr/include/python$(PYTHON_VERSION) \
 		$(1)/usr/include/
 	$(CP) \
-		$(HOST_PYTHON_LIB_DIR) \
+		$(PKG_INSTALL_DIR)/usr/lib/python$(PYTHON_VERSION) \
 		$(PKG_INSTALL_DIR)/usr/lib/libpython$(PYTHON_VERSION).so* \
 		$(1)/usr/lib/
 	$(CP) \
-		$(HOST_PYTHON_DIR)/lib/pkgconfig/python.pc \
-		$(HOST_PYTHON_DIR)/lib/pkgconfig/python2.pc \
-		$(HOST_PYTHON_DIR)/lib/pkgconfig/python-$(PYTHON_VERSION).pc \
+		$(PKG_INSTALL_DIR)/usr/lib/pkgconfig/python*.pc \
 		$(1)/usr/lib/pkgconfig
-	$(CP) \
```

Comment:
i think this `/config` folder is important;
i would leave it if possible

Reply:
Because I changed it to copy `$(PKG_INSTALL_DIR)/usr/lib/python$(PYTHON_VERSION)` (instead of `$(HOST_PYTHON_LIB_DIR)`) a few lines earlier, it already includes the `config` folder :joy:

Reply:
oh; ok :)

thanks for the info